gensim/models/atmodel.py

```diff
@@ -354,7 +350,13 @@ def extend_corpus(self, corpus):
             self.corpus.extend(corpus)
 
     def compute_phinorm(self, expElogthetad, expElogbetad):
-        """Efficiently computes the normalizing factor in phi."""
+        """Efficiently computes the normalizing factor in phi.
+
+        Parameters
+        ----------
+        expElogthetad: numpy.ndarray
```

Comment:
where is parameter description?

Reply:
I wasnt sure about what to write for the description, which is why I left it blank.